catalogData/elasticsearch24/3x-ha/k8s/statefulset-data.json

```diff
@@ -151,7 +151,7 @@
                 }
               }
             },
-            "imagePullPolicy": "Always"
+            "imagePullPolicy": "IfNotPresent"
```

Comment:
What impact does this have on our daily rebuild? Based on https://kubernetes.io/docs/concepts/configuration/overview/#container-images, I'm thinking we'll never re-pull the image with the `2.4.4` tag, even if it's changed at the registry. Is this where we bump the image tag via environment variable like you suggested earlier?